frontend/src/metabase/entities/containers/rtk-query/types/index.ts

```diff
@@ -0,0 +1,64 @@
+import type { BaseQueryFn, QueryDefinition } from "@reduxjs/toolkit/query";
```

Comment:
Entity framework internals meet TypeScript for the first time.
Beware: types in this file are incomplete and may be inaccurate.